lib/statesman/adapters/active_record.rb

```diff
@@ -125,6 +121,21 @@ def transition_conflict_error?(e)
         e.message.include?(@transition_class.table_name) &&
           (e.message.include?("sort_key") || e.message.include?("most_recent"))
       end
+
+      def updated_timestamp_options
+        timestamp = if ::ActiveRecord::Base.default_timezone == :utc
+                      Time.now.utc
+                    else
+                      Time.now
+                    end
+
+        updated_timestamp_columns.
+          each_with_object({}) { |column_name, options| options[column_name] = timestamp }
+      end
+
+      def updated_timestamp_columns
+        @transition_class.send(:timestamp_attributes_for_update_in_model)
```

Comment:
It might be a terrible idea to depend on this private method. Other options include:

* Using the public `column_names` method, and assuming that *we just know* what the column names we should be interested in are (i.e. hard-coding `updated_at` and `updated_on`)
* Requiring that you have an `updated_at` column (thereby adding additional requirements over and above Rails)
* Making the updated timestamp column somehow configurable (thereby duplicating Rails' config)